stablehlo/tests/CheckOps.cpp

```diff
@@ -128,6 +136,29 @@ llvm::Error evalExpectSerializedEqOp(const Tensor &expected, StringRef probeId,
   return evalExpectEqOp(expected, *tensorOrError);
 }
 
+llvm::Error evalCustomCallCheckEq(stablehlo::CustomCallOp op,
```

Comment:
Similarly, let's keep this custom call handling in `StablehloTranslateMain` as it was. The custom call is a bit of a hack for `testdata` testing, it shouldn't be used more broadly than our tests. A better solution would be adding a pass that converts custom_call -> check dialect, but no need to do it now